src/Particle/ParticleSpatialLayout.hpp

```diff
@@ -114,13 +114,12 @@ namespace ippl {
 
         int tag = Comm->next_tag(mpi::tag::P_SPATIAL_LAYOUT, mpi::tag::P_LAYOUT_CYCLE);
```

Comment:
I am not entirely sure what exactly that is used for but from what I can see, this can be used to generate MPI tags for communication and not for the buffers. Maybe @matt-frey can comment on this.

Following this particular variable you mentioned, I can see that [this](https://github.com/JoMee/ippl_fork/blob/buffer-refactoring/src/Particle/ParticleBase.h#L292) function is called.